apps/api/src/config/env.config.ts

```diff
@@ -1,9 +1,18 @@
-import { EnvConfig } from "src/interfaces/env.config.interface";
+import { EnvConfig } from 'src/interfaces/env.config.interface';
 
 export default (): EnvConfig => ({
   port: parseInt(process.env.API_PORT, 10) || 8000,
+  mongoUri: process.env.MONGO_URI,
   auth0: {
+    clientId: process.env.AUTH0_CLIENT_ID,
     domain: process.env.AUTH0_DOMAIN,
     audience: process.env.AUTH0_API,
   },
+  aws: {
+    client: {
+      accessKey: process.env.AWS_ACCESS_KEY,
+      secretKey: process.env.AWS_SECRET_KEY,
+    },
+    s3Bucket: process.env.AWS_S3_BUCKET,
+  },
```

Comment:
You can destructure the `process.env` variable and you can save some typing time and read, like:

````javascript
const { MONGO_URI, AUTH0_CLIENT_ID, ..., AWS_S3_BUCKET } = process.env;

export default (): EnvConfig => ({
  port: parseInt(API_PORT, 10) || 8000,
  mongoUri: MONGO_URI,
  auth0: {
    clientId: AUTH0_CLIENT_ID,
  ....
  s3Bucket: AWS_S3_BUCKET,
  },
 ....
 ````

Reply:
@eKoh951 When I do that the properties are `undefined`, I don't know why, I was investigating a lot and I didn't find a solution, so for now I leave it as it was before